test/tf/packet/main.tf

```diff
@@ -0,0 +1,19 @@
+provider "packet" {}
+
+resource "packet_project" "project" {
+  name = "go-discover:packet"
```

Comment:
Can you add a bit of randomness here? See vsphere for [an example](https://github.com/hashicorp/go-discover/blob/master/test/tf/vsphere/main.tf#L89-L98). If this is a unique name in Packet that will help the tests from conflicting with each other.